src/public/modules/forms/services/spcp-session.client.factory.js

```diff
@@ -36,20 +36,11 @@ function SpcpSession($window, $cookies) {
     clearUserName: function () {
       session.userName = undefined
     },
-    logout: function () {
-      $cookies.remove(
-        session.cookieName,
-        $window.spcpCookieDomain ? { domain: $window.spcpCookieDomain } : {},
-      )
+    logout: function (authType) {
+      PublicFormAuthService.spcpLogout(authType)
```

Comment:
do we need to use `$q.when` and a `catch` here? what if the logout is unsuccessful?

Reply:
edited